apps/projection/src/styles.scss

```diff
@@ -1,3 +1,15 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+app-card {
+  .bg-light-red {
+    background-color: rgba(250, 0, 0, 0.1);
+  }
+  .bg-light-green {
+    background-color: rgba(0, 250, 0, 0.1);
+  }
+  .bg-light-yellow {
+    background-color: rgba(230, 234, 12, 0.301);
+  }
+}
```

Comment:
When the app is small, that's ok. but It will create a bloated file. and the style are far from where there are used. prefer to keep them as close as their component. 

only global style should come here